wolfcrypt/src/asn.c

```diff
@@ -17022,7 +17036,7 @@ static int DecodeConstructedOtherName(DecodedCert* cert, const byte* input,
     if (ret == 0 && dnsEntry != NULL) {
         dnsEntry->type = ASN_OTHER_TYPE;
         dnsEntry->len = strLen;
-        dnsEntry->name = (char*)XMALLOC(strLen + 1, cert->heap,
+        dnsEntry->name = (char*)XMALLOC((word32)strLen + 1, cert->heap,
```

Comment:
is doing a cast to word32 on the malloc size portable enough? i.e on Ubuntu and with wolfSSL_Malloc we use size_t type.

Reply:
:heavy_check_mark: